include/audio_defines.h

```diff
@@ -102,4 +103,35 @@ struct audio_data {
 	struct audio_metadata meta;
 };
 
+/**
+ * @brief Get the number of channels in the audio data.
+ *
+ * This function will count the number of bits set in the
+ * locations field of the audio metadata.
+ *
+ * @param audio_frame Pointer to the audio data structure.
+ *
+ * @return The number of channels.
+ */
+static inline uint8_t audio_data_num_ch_get(struct audio_data const *const audio_frame)
```

Comment:
Suggest to use `bt_audio_get_chan_count` in this function

Reply:
This has been done